drivers/ovsdriver.go

```diff
@@ -432,8 +432,8 @@ func (d *OvsDriver) DeleteMaster(node core.ServiceInfo) error {
 	return nil
 }
 
-// AddBgpNeighbors adds bgp neighbor by named identifier
-func (d *OvsDriver) AddBgpNeighbors(id string) error {
+// AddBgp adds bgp config by named identifier
+func (d *OvsDriver) AddBgp(id string) error {
```

Comment:
curious why is this code in ovsdriver.go
I mean we should avoid keeping non driver logic in ovsdriver.go; this would help other drivers to leverage this code e.g. @danehans request to do ipvlan driver